src/renderers/controls/text.control.tsx

```diff
@@ -36,6 +40,9 @@ export class TextControl extends Control<ControlProps, ControlState> {
                hidden={!visible}
                disabled={!enabled}
                autoFocus={uischema.options && uischema.options.focus}
+               maxlength={uischema.options && uischema.options.restrict ? maxLength : undefined}
+               size={uischema.options && uischema.options.trim ? maxLength : undefined}
+               style={uischema.options && uischema.options.trim ? 'width: initial' : ''}
```

Comment:
why do we need a style attribute in the case of a normal control?

Reply:
You are right, I confused the two. When testing, I temporarily removed the withIncreasedRank(...) from materialize to display the normal control. But then I forgot to select Style "None" from Dropdown menu.

I will fix that, but my fix needs more than removing just the style. In the example, the input field stretches over the entire width of the screen (or its div container, respectively), so somehow the size attribute gets overwritten.